docs/02.API-REFERENCE.md

```diff
@@ -2994,6 +3020,48 @@ jerry_get_value_from_error (jerry_value_t value, bool release)
 - [jerry_create_error_from_value](#jerry_create_error_from_value)
 - [jerry_create_abort_from_value](#jerry_create_abort_from_value)
 
+## jerry_set_error_object_update_callback
+
+**Summary**
+
+Set the decorator callback for newly created Error objects. The operation of the callback
+is described in [jerry_error_object_update_callback_t](#jerry_error_object_update_callback_t).
+
+**Prototype**
+
+```c
+void jerry_set_error_object_update_callback (jerry_error_object_update_callback_t callback);
+```
+
+- `callback` - callback function, the previously set value is overwritten, and setting NULL
+               disables the operation
+
+*New in version [[NEXT_RELEASE]]*.
+
+**Example**
+
+```c
+static void
+error_object_update_callback (const jerry_value_t error_object) /**< new error object */
+{
+  (void) error_object;
+  printf ("Notification: a new error is created\n");
+}
+
+void main(void)
+{
+  jerry_init (JERRY_INIT_EMPTY);
+
+  jerry_set_error_object_update_callback (error_object_update_callback);
+
```

Comment:
I'd manually create a new error after the callback is set.

Reply:
Manuallly?

Reply:
I meant to simply create a new error value.